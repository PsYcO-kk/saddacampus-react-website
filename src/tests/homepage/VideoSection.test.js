import {shallow} from 'enzyme';
import React from 'react';
import VideoSection from '../../components/homepage/VideoSection';

test("should render VideoSection correctly" , ()=>{
    const wrapper = shallow(<VideoSection />)
    expect(wrapper).toMatchSnapshot();
})
