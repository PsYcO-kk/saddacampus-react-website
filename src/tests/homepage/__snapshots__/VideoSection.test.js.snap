// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render VideoSection correctly 1`] = `
<section
  className="home-about-area section-gap"
>
  <div
    className="container"
  >
    <div
      className="row align-items-center"
    >
      <div
        className="col-lg-6 about-left"
      >
        <video
          className="embed-responsive embed-responsive-16by9"
          controls={true}
          poster="/assets/images/one-o-16-9.jpeg"
          preload="metadata"
        >
          <source
            src="/assets/videos/one-o.mp4"
            type="video/mp4"
          />
          Your browser does not support the video tag.
        </video>
      </div>
      <div
        className="col-lg-6 about-right"
      >
        <h1>
          Our first strive to fulfill your HUNGER FOR HAPPINESS
        </h1>
        <h4>
          Launching Saddacampus One.0
        </h4>
        <p>
          In the month of March, the evening scenery of the IIT (ISM) Main Gate changed owing to the presence of Saddacampus One.0 providing eat in, take away and hostel delivery at the best prices ever.
        </p>
        <a
          className="text-uppercase primary-btn"
          href="#"
        >
          Get Details
        </a>
      </div>
    </div>
  </div>
</section>
`;
